server/portal/apps/webhooks/migrations/0001_squashed_0003_alter_externalcall_callback_data.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.19 on 2023-05-04 22:02
```

Comment:
After these squashed migrations are applied to all deployed v2 portals, we can remove the migrations that were squashed underneath it.

https://docs.djangoproject.com/en/3.2/releases/3.1/#jsonfield-for-all-supported-database-backends